### DISCUSSION TITLE
Trouble getting corner to corner linear gradient

### DISCUSSION THREAD

**Discussion by Anequit** [AvaloniaUI/Avalonia#7494] [Q&A]
I can only seem to get either vertical or horizontal, but not corner to corner. Can someone provide a corner to corner example?

**Comment by amwx** [comment]
Set the Start and End points to the corners of whatever is hosting the gradient, in this case I'm using relative points (the %) to draw from the top left to the bottom right. 

````
<Border Width="100" Height="100">
    <Border.Background>
        <LinearGradientBrush StartPoint="0%,0%" EndPoint="100%,100%">
            <!-- define gradient stops here -->
        </LinearGradientBrush>
    </Border.Background>
</Border>
````
